sdk/docs/core/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 
 ## 1.0.0-preview.4 (Unreleased)
 
+- API breaking changes:
+  - `az_span.h`:
+    - `az_span_init(...)` is renamed to `az_span_create(...)`
+    - `az_span_from_str(...)` is renamed to `az_span_create_from_str(...)`
```

Comment:
I used periods at the end of each item. Please make them to match after merging my changes. I slightly prefer periods at the end, because if we add a 2-sentence item, we end up with 2 periods, ant there's a slight mismatch with 1-sentence items. But I've seen everywhere (not only our repo) it done both ways, and it is minor..